jsk_recognition_utils/CMakeLists.txt

```diff
@@ -42,6 +43,17 @@ catkin_package(
  CATKIN_DEPENDS jsk_recognition_msgs pcl_ros visualization_msgs message_runtime
 )
 
+catkin_generate_virtualenv(
+  # Select an alternative major version of the python interpreter - it must be installed on the system.
+  PYTHON_VERSION_MAJOR 2  # Default 2
+
+  # Choose not to use underlying system packages. This excludes any python packages installed by apt or system-pip from the environment.
+  USE_SYSTEM_PACKAGES TRUE  # Default TRUE
+
+  # Disable including pip requirements from catkin dependencies of this package.
+  ISOLATE_REQUIREMENTS FALSE  # Default FALSE
+)
```

Comment:
Are these comments necessary?

Reply:
Thanks for review!
Removed unnecessary comments.